nginx/nginx.go

```diff
@@ -221,25 +267,22 @@ func (lb *nginxLoadBalancer) Stop() error {
 	return lb.lastErr.Get()
 }
 
+// This is called by a single go routine from the controller
 func (lb *nginxLoadBalancer) Update(entries controller.IngressUpdate) error {
 	updated, err := lb.update(entries)
 	if err != nil {
 		return fmt.Errorf("unable to update nginx: %v", err)
 	}
 
 	if updated {
-		err = lb.signaller.sighup(lb.cmd.Process)
-		if err != nil {
-			return fmt.Errorf("unable to signal nginx to reload: %v", err)
-		}
-		log.Info("Nginx updated")
+		lb.signalRequired()
+		log.Infof("Nginx update required (can take up to %d seconds)", lb.Conf.UpdateFrequencySeconds)
```

Comment:
Not sure this is worthwhile logging - if we put a log message when it does the actual update. We'll already get lots of log spam when an update comes in.